### ISSUE TITLE
location of post.js and post.css changed in post.php

### ISSUE THREAD

**Issue by dustinbrisebois** [catalyst/moodle-auth_saml2#204]
When using the saml login goes to a white page with "submit" in corner, this is due to the post.js and post.css files being relocated but not having their urls updated in:

extlib\simplesamlphp\templates\post.php

in post.php on lines 4 to 10 should be:

```
<head>
    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
    <title>POST data</title>
    <script type="text/javascript" src="/<?php echo $this->data['baseurlpath']; ?>../extlib/simplesamlphp/www/resources/post.js"></script>
    <link
        type="text/css" rel="stylesheet" href="/<?php echo $this->data['baseurlpath']; ?>../extlib/simplesamlphp/www/resources/post.css" />
</head>
```

As post.js and post.css have been moved from the previous folder and point to the new location.

**Comment by roperto** [comment]
Hi @dustinbrisebois , I cannot reproduce this issue. If I just update those lines they will be lost next time we update the libs, so we need a testcase for it.

Can you provide more details on your configuration?

Thanks

**Comment by JeromeLam** [comment]
The following line will produce https://MY.WEBSITE.COM/auth/saml2/sp/resources/post.js, which does not exist:
https://github.com/catalyst/moodle-auth_saml2/blob/261c132657cb85866050b7c992b3eee31507acf3/extlib/simplesamlphp/templates/post.php#L7

Here's where the configuration of **baseurlpath** is:
https://github.com/catalyst/moodle-auth_saml2/blob/261c132657cb85866050b7c992b3eee31507acf3/config/config.php#L44

If it makes sense, config.php#L44 should be updated to
`'baseurlpath' => $wwwroot . '/auth/saml2/extlib/simplesamlphp/www/'`.

**Comment by roperto** [comment]
The suggested fix has been applied, but I am reopening this do add the proper documentation/testcase and until released on plugin directory.

**Comment by mjthompson** [comment]
Thank you so much for this report @dustinbrisebois - we updated the plugin to the latest version and encountered this issue. Making those two reference changes fixed the issue.

**Comment by roperto** [comment]
Hi. This has been fixed in 2018032600

Please let me know if you are still having any issues.
